src/main/java/core/basesyntax/strategy/BalanceOperationHandler.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax.strategy;
+
+import core.basesyntax.model.Transaction;
+import core.basesyntax.storage.Storage;
+
+public class BalanceOperationHandler implements OperationHandler {
+    @Override
+    public void process(Transaction transaction) {
+        Integer initialQuantity = Storage.storage.get(transaction.getFruit());
+        Storage.storage.put(transaction.getFruit(),
+                initialQuantity == null

```

Comment:
1. This check is redundant, 
2. At the same time, in some case we can get incorrect data, so we don`t need to sum balance and initial quantity,
we shoul just put balance value to our HashMap